project-base/src/Shopsys/ShopBundle/Model/Payment/PaymentDataFactory.php

```diff
@@ -4,31 +4,12 @@
 
 namespace Shopsys\ShopBundle\Model\Payment;
 
-use Shopsys\FrameworkBundle\Component\Domain\Domain;
-use Shopsys\FrameworkBundle\Component\Image\ImageFacade;
 use Shopsys\FrameworkBundle\Model\Payment\Payment as BasePayment;
 use Shopsys\FrameworkBundle\Model\Payment\PaymentData as BasePaymentData;
 use Shopsys\FrameworkBundle\Model\Payment\PaymentDataFactory as BasePaymentDataFactory;
-use Shopsys\FrameworkBundle\Model\Payment\PaymentFacade;
-use Shopsys\FrameworkBundle\Model\Pricing\Vat\VatFacade;
 
 class PaymentDataFactory extends BasePaymentDataFactory
 {
-    /**
-     * @param \Shopsys\FrameworkBundle\Model\Payment\PaymentFacade $paymentFacade
-     * @param \Shopsys\FrameworkBundle\Model\Pricing\Vat\VatFacade $vatFacade
-     * @param \Shopsys\FrameworkBundle\Component\Domain\Domain $domain
-     * @param \Shopsys\FrameworkBundle\Component\Image\ImageFacade $imageFacade
-     */
-    public function __construct(
```

Comment:
Why have you removed this?

Reply:
Why not? It is just constructor who call parent construcotr, not it? And in parent add service (CurrencyFacade) and I would have to add servic  in this class too. So I think that delete this constructar was better

Reply:
So extension is easier. You should update this constructor and add such info to upgrade notes

Reply:
Ok, i got it back